src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+
+const Breadcrumbs = (props) => {
+  const {
+    history,
+    location: { pathname },
+  } = props;
+  const pathnames = pathname.split("/").filter((x) => x);
+  return (
+    <div className="avenir pa2 ph5 link flex">
+      {pathnames.length > 0 && (
```

Comment:
I'd suggest moving `pathnames.length > 0` to the top level so it won't take extra space when there's no path

Reply:
Yup, the code could be a bit easier to read if you assigned the different high level parts to variables. I think this part is the home crumb?

Reply:
> I'd suggest moving `pathnames.length > 0` to the top level so it won't take extra space when there's no path

There was no extra space if there was no path. To verify, i tried changing the background color of the breadcrumbs and nothing was rendered if there's no path. 

Nevertheless, i reorganized/simplified the logic and put the conditional statement at top level per suggestions.